tests/unit/scanners/test.javascript.js

```diff
@@ -144,6 +125,39 @@ describe('JavaScript Scanner', () => {
     expect(linterMessages.length).toEqual(0);
   });
 
+  it('should scan node modules', async () => {
+    const code = 'el.innerHTML = evilContent';
+
+    const jsScanner = new JavaScriptScanner(
+      code,
+      'node_modules/module/code.js'
+    );
+
+    const { linterMessages } = await jsScanner.scan();
+    expect(linterMessages[0]).toMatchObject({ code: 'UNSAFE_VAR_ASSIGNMENT' });
```

Comment:
@willdurand thanks for making this assertion a bit more explicit! :heart: 

lgtm, r+ renewed